content/en/docs/faq/byop.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Bring your own Prometheus"
+keywords: "Monitoring, Prometheus, node-exporter, kube-state-metrics, KubeSphere, Kubernetes"
+description: "Use your own Prometheus stack for KubeSphere monitoring"
+
+Weight: 7100
+---
+
+KubeSphere comes with several pre-installed customized monitoring components including Prometheus Operator, Prometheus, Alertmanager, Grafana (Optional), various service monitors, node-exporter, kube-state-metrics. These components might already exist before you install KubeSphere, it's possible to use your own Prometheus stack setup in KubeSphere v3.0.0 .
```

Comment:
Just one thing to confirm here. Regards these components mentioned in this paragraphs, will they be automatically installed if KubeSphere is installed with the minimum package. Do I need to enable components like logging or alerting to have them installed?

Reply:
they will be automatically installed with the minimum package